_layouts/base.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="{{ page.lang | default: site.lang | default: 'en-US' }}">
+  <head>
+    {% include meta.html %}
+    {% include styles.html %}
+    {% include google_analytics.html %}
+    {% include dap.html %}
+    {% include google_recaptcha.html %}
+  </head>
+  <body
+    class="{{ layout.class }} {{ page.class }} site lang-{{ site.lang }}"
+    {% if layout.body.spy %}data-spy="{{ layout.body.spy }}"{% endif %}
+    {% if layout.body.target %}data-target="#{{ layout.body.target }}"{% endif %}
```

Comment:
```suggestion
    {% if layout.body.target %}data-target="{{ layout.body.target }}"{% endif %}
```

Reply:
(`#` was being duplicated because `target` is passed as `#pb-nav--side-cntnr` from the layout)

Reply:
oh great catch, thanks!